pages/dkp/kommander/2.2/release-notes/platform-application-versions/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: layout.pug
+navigationTitle: Supported Platform Applications
+title: Supported Platform Applications
+menuWeight: 60
+excerpt: Upgrade Platform applications
+beta: false
+---
+ 
+<!-- markdownlint-disable MD0013 MD030 MD034 -->
+
+Upgrading to the latest version of DKP adapts your Platform Applications to the following versions:
+
+| Platform Application in 2.x | Platform Application in 2.2 |
+| --- | --- |
+|  |  |
+| cert-manager-setup | cert-manager-setup |
+| https://mesosphere.github.io/charts/stable/index.yaml | https://mesosphere.github.io/charts/stable/index.yaml |
+| 0.2.7 | 0.2.7 |
```

Comment:
We need to work with engineering to get the underlying component version numbers here instead of these 'Kommander application' version numbers which are more or less useless for anyone other than a D2iQ developer.    I.e. this 0.2.7 is an internal number we create which refers to the version of the chart, not which version of the 'Cert-Manager' component is installed.

Reply:
I'd also like to see, in the release notes, any updated components in that list.

Reply:
agreed, proposed new table format in above comment that would include any underlying component versions